opae.spec

```diff
@@ -183,15 +170,31 @@ popd
 %license %{_datadir}/opae/COPYING
 
 %{_libdir}/libopae-c.so.%{version}
-%{_libdir}/libopae-c.so.1
+%{_libdir}/libopae-c.so.2
```

Comment:
Why not use .* on these so that they don't require further updates when the version changes?

Reply:
https://docs.fedoraproject.org/en-US/packaging-guidelines/
https://github.com/trixirt/opae-rpm/blob/opae-1.4.0-5/opae.spec

Tom (Red hat) was updated fedora Spec file in opae 1.4.0 release, followed  fedora spec guidelines

Reply:
Seems like a maintenance issue to me.  I doubt there's anything in the guidelines that says you have to do things inefficiently.

Reply:
-%{_libdir}/libopae-c.so.%{version}
-%{_libdir}/libopae-c.so.2
-%{_libdir}/libbitstream.so.%{version}
-%{_libdir}/libbitstream.so.2
-%{_libdir}/libopae-cxx-core.so.%{version}
-%{_libdir}/libopae-cxx-core.so.2
-%{_libdir}/libopae-c++-utils.so.%{version}
-%{_libdir}/libopae-c++-utils.so.2
-%{_libdir}/libopae-c++-nlb.so.%{version}
-%{_libdir}/libopae-c++-nlb.so.2
-%{_libdir}/libfpgad-api.so.%{version}
-%{_libdir}/libfpgad-api.so.2
-
+%{_libdir}/libopae-c.so.*
+%{_libdir}/libbitstream.so.*
+%{_libdir}/libopae-cxx-core.so.*
+%{_libdir}/libopae-c++-utils.so.*
+%{_libdir}/libopae-c++-nlb.so.*
+%{_libdir}/libfpgad-api.so.*

seems to be ok wrt rpmlint.
So glob away!

Reply:
fixed , updated Spec file with module name .so.*  ( glob)